docker.go

```diff
@@ -366,11 +390,6 @@ func commandBuild(build Build) *exec.Cmd {
 			args = append(args, "--label", label)
 		}
 	}
-
-	// we need to enable buildkit, for secret support and ssh agent support
-	if build.Secret != "" || len(build.SecretEnvs) > 0 || len(build.SecretFiles) > 0 || build.SSHAgentKey != "" {
-		os.Setenv("DOCKER_BUILDKIT", "1")
```

Comment:
Is this set globally somewhere when buildx is used?

Reply:
NA anymore.